src/charm.py

```diff
@@ -469,20 +469,24 @@ def _snap_plugs_logging_configs(self) -> List[Dict[str, Any]]:
         agent_fstab = SnapFstab(Path("/var/lib/snapd/mount/snap.grafana-agent.fstab"))
 
         shared_logs_configs = []
-        for endpoint in self._cos.snap_log_endpoints:
-            fstab_entry = agent_fstab.entry(endpoint.owner, endpoint.name)
+        endpoint_owners = {endpoint.owner for endpoint in self._cos.snap_log_endpoints}
+        for fstab_entry in agent_fstab.entries:
+            if fstab_entry not in endpoint_owners:
```

Comment:
Shouldn't this be like this?
```suggestion
            if fstab_entry.owner not in endpoint_owners:
```

Reply:
Oh yes, I noticed it but forgot to push the commit :P